From: Ole Streicher <olebole@debian.org>
Date: Fri, 14 Oct 2016 17:09:23 +0200
Subject: Add local Eigen3 include dir

This is required since we use a locally patched version of Eigen3. See
debian/README.source for details.
---
 cmake_files/dependencies.cmake | 2 +-
 cpp/CMakeLists.txt             | 1 -
 cpp/purify/CMakeLists.txt      | 1 -
 3 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/cmake_files/dependencies.cmake b/cmake_files/dependencies.cmake
index 4df4b1a..9887810 100644
--- a/cmake_files/dependencies.cmake
+++ b/cmake_files/dependencies.cmake
@@ -36,7 +36,7 @@ else()
   find_package(Boost REQUIRED)
 endif()
 
-find_package(Eigen3 REQUIRED)
+include_directories(${CMAKE_SOURCE_DIR}/eigen3)
 
 if(logging)
   find_package(spdlog REQUIRED)
diff --git a/cpp/CMakeLists.txt b/cpp/CMakeLists.txt
index c528ac7..6544533 100644
--- a/cpp/CMakeLists.txt
+++ b/cpp/CMakeLists.txt
@@ -17,7 +17,6 @@ function(add_include_dir)
 endfunction()
 
 add_include_dir(
-  ${EIGEN3_INCLUDE_DIR}
   ${Boost_INCLUDE_DIR}
   ${CFitsIO_INCLUDE_DIR}
   ${CCFits_INCLUDE_DIR}/..
diff --git a/cpp/purify/CMakeLists.txt b/cpp/purify/CMakeLists.txt
index edceff2..f500225 100644
--- a/cpp/purify/CMakeLists.txt
+++ b/cpp/purify/CMakeLists.txt
@@ -40,7 +40,6 @@ target_include_directories(libpurify PUBLIC
 
 
 add_include_dir(
-  ${EIGEN3_INCLUDE_DIR}
   ${Boost_INCLUDE_DIR}
   ${CFitsIO_INCLUDE_DIR}
   ${CCFits_INCLUDE_DIR}/..
